src/node/node_state.h

```diff
@@ -596,6 +599,282 @@ namespace ccf
         std::chrono::milliseconds(config.joining.join_timer));
     }
 
+    template <typename T>
+    void send_refresh_jwt_keys(T msg)
+    {
+      auto body = serdes::pack(msg, serdes::Pack::Text);
+
+      http::Request request(fmt::format(
+        "/{}/{}",
+        ccf::get_actor_prefix(ccf::ActorsType::members),
+        "jwt_keys/refresh"));
+      request.set_header(
+        http::headers::CONTENT_TYPE, http::headervalues::contenttype::JSON);
+      request.set_body(&body);
+      // Need a custom authentication policy that accepts only node certs.
+      // See https://github.com/microsoft/CCF/issues/1904
+      // http::sign_request(request, node_sign_kp);
```

Comment:
I might be missing something but if the signed request doesn't need to be recorded, presenting the `node_cert` in the `enclave::SessionContext` is probably sufficient to authenticate the node? A client wouldn't be able to present the node certificate in the TLS session as it would also require presenting the node private key which never leaves the node's enclave.

Reply:
Yeah you may be right. If I keep auth disabled but then check the cert in the endpoint manually then that's probably good enough.

Reply:
Let's defer this until #1904 is done.